miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditBoxController.java

```diff
@@ -171,9 +192,43 @@ public String processSubmit(@ModelAttribute("box") Box box, ModelMap model, Sess
       Box original = boxService.get(box.getId());
       box.setBoxables(original.getBoxables());
     }
+    log.info("" + box.getStorageLocation().getId());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)